### DISCUSSION TITLE
[Poseidon] Feature Request - Slim Menu

### DISCUSSION THREAD

**Discussion by mekhaba** [primefaces/community#49] [PrimeFaces Templates] [answered]
is there any update for slim mode menu ? 

I hope so.

**Comment by atkntepe** [comment, marked as answer]
Hi, at the moment, we dont have a plan for adding a slim menu to Posedion Template. We are planning a new chapter for our templates. We will change some of the templates with overall makeup and improvements. We will make sure to announce these plans when it is definitive. 

At the moment, I don't know if the Poseidon Template will be in this new list of templates. But if it does, I will let you know. If you have you have any other questions, feel free to ask me here. 👍
